### ISSUE TITLE
Error with AUCell.buildRankings

### ISSUE THREAD

**Issue by asmagen** [aertslab/SCENIC#2]
Hello,

While trying to reproduce the results found in the tutorials here I find that the 'AUCell.buildRankings' function does not exist and therefore I can't run the complete tutorials code.

> library(AUCell)
> aucellRankings <- AUCell.buildRankings(exprMat, nCores=4, plotStats=TRUE)
Error: could not find function "AUCell.buildRankings"

I made sure I have the last version of the package installed:
> devtools::install_github("aertslab/AUCell", build_vignettes=F)
Skipping install of 'AUCell' from a github remote, the SHA1 (568aa712) has not changed since last install.

How can it be fixed?

Thanks,
Assaf

**Comment by s-aibar** [comment]
Hello,

That AUCell function has been renamed to AUCell_buildRankings (requirement to include it in Bioconductor). 

SCENIC will be updated accordingly, but the current version of SCENIC is meant to use the following versions of AUCell and RcisTarget:

- AUCell_0.99.5.tar.gz (http://scenic.aertslab.org/downloads/Rpackages/AUCell_0.99.5.tar.gz)
- RcisTarget_0.99.0.tar.gz (http://scenic.aertslab.org/downloads/Rpackages/RcisTarget_0.99.0.tar.gz)

**Comment by asmagen** [comment]
Thanks.

However now getAuc doesn't work. Is this also a result of recent changes to the package? It makes a practical use of this method and tutorials beyond reach.

Is there a tutorial that's consistent with the changes made to the package?

Thank you

**Comment by s-aibar** [comment]
If you are running SCENIC (current version: 0.1.5), you should use AUCell version 0.99.5 (see link above). Otherwise, it will indeed be hard to follow the tutorials. 

As these are still development versions, several functions of the package have been modified. If you install the latest version of AUCell, then you should use the tutorial in that given package (those should be consistent, if you find something that doesn't match there, please let me know), but be aware that some functions in SCENIC might have been renamed. We will update SCENIC package once we have the final version of AUCell, GENIE3 & RcisTarget (to avoid having even more versions...).

Sorry for the inconvenience,
Sara

**Comment by julien-roux** [comment]
I confirm that the `AUCell.buildRankings` function is not exported leading to an error.
However the following command works fine for me:
```{r}
library(AUCell)
library(SCENIC)
aucellRankings <- AUCell::AUCell.buildRankings(exprMat, nCores=4, plotStats=TRUE)
```
